localstack/services/sns/provider.py

```diff
@@ -113,12 +110,7 @@ def _get_topic(arn: str, context: RequestContext) -> Topic:
     def get_topic_attributes(
         self, context: RequestContext, topic_arn: topicARN
     ) -> GetTopicAttributesResponse:
-        try:
-            moto_response: GetTopicAttributesResponse = call_moto(context)
-        except CommonServiceException as exc:
-            if exc.code == "NotFound":
-                raise NotFoundException("Topic does not exist")
-            raise
+        moto_response: GetTopicAttributesResponse = call_moto(context)
```

Comment:
question: why are we removing a fix to then add a `skip_snapshot_verify` path? This was done to counter the specific issue in the test? ` # This undocumented field is currently not returned by Moto`

Reply:
This is now implemented in Moto upstream, so the ignore has now been removed

https://github.com/getmoto/moto/pull/6465

Reply:
Awesome, thanks for fixing it upstream! that's better than my solution 😄